fiat-api/src/main/java/com/netflix/spinnaker/config/FiatAuthenticationConfig.java

```diff
@@ -58,14 +62,17 @@
   @Setter
   private OkClient okClient;
 
-  @Value("${services.fiat.baseUrl}")
-  @Setter
-  private String fiatEndpoint;
+  @Bean
+  @ConfigurationProperties("services.fiat")
+  public FiatConfigurationProperties fiatConfigurationProperties() {
+    return new FiatConfigurationProperties();
+  }
```

Comment:
Instead of defining the whole bean here, you can simply put this at the top..

```
@EnableConfigurationProperties(FiatConfigurationProperties)
```

Put @ConfigurationProperties("services.fiat") on the class definition for the config prop class.